Data/Sys/GameSettings/NAL.ini

```diff
@@ -19,5 +19,9 @@ StereoConvergence = 5000
 # This game creates a large number of EFB copies at different addresses, resulting
 # in a large texture cache which takes considerable time to save.
 SaveTextureCacheToState = False
-# This game crashes very soon after boot if above 1x IR
-InternalResolution = 1
+
+[Video_Hacks]
+# This game crashes very soon after boot if EFB copies are scaled at higher resolutions.
+# This is due to Dolphin running out of memory.  Disabling Scaled EFB copies allows playing
+# at higher resolutions without the game crashing for now.
+EFBScaledCopy = True
```

Comment:
Do you mean `False` here?  The comment and the value don't match I think

Reply:
I didn't realize I never pushed this - should be fixed now.